### ISSUE TITLE
Separate Redo Files

### ISSUE THREAD

**Issue by tenplus1** [acmgit/cucina_vegana#27]
Farming Redo is fully farming API compliant so having two versions for each crop isn't required, the default version will work totally fine with Farming Redo.

**Comment by acmgit** [comment]
Have you tested it?

**Comment by tenplus1** [comment]
Yes it has been tested and works fine, the default game's farming api works in farming redo as well.

**Comment by acmgit** [comment]
Minetest without farming_redo, the seeds named: seed_xxxxx. (xxxx is the name of the crop.)
In farming_redo, the seeds are named: xxxxx_seed.

Minetest's registration of new crops works with one declaration, where you add the number of steps.
In farming_redo, you have to register each stage separate.
